classes/Twig/RestoreSteps.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace PrestaShop\Module\AutoUpgrade\Twig;
+
+use PrestaShop\Module\AutoUpgrade\UpgradeTools\Translator;
+
+class RestoreSteps
+{
+    const STEP_BACKUP_SELECTION = 'backup-selection';
+    const STEP_RESTORE = 'restore';
+    const STEP_POST_RESTORE = 'post-restore';
+
+    /** @var Translator */
+    private $translator;
+
+    public function __construct(Translator $translator)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * @return array<self::STEP_*, array<string,string>>
+     */
+    public function getSteps(): array
```

Comment:
An interface would be useful to enforce the same method signature between `RestoreSteps` and `UpdateSteps`